etc/schema.json

```diff
@@ -212,6 +212,10 @@
                 "xff": {
                     "type": "string"
                 },
+                "extra": {
+                    "type": "object",
```

Comment:
I'm trying to get a description on everything new for documentation purposes, can you add one?